Spec.md

```diff
@@ -274,7 +274,7 @@ three source directives:
 In addition a destination directive *must* be provided:
 
 - `install_to`: The location where this section should be installed.
-  Valid values for this entry are `GameData`, `Ships`, `Ships/SPH`(**v1.12**), `Ships/VAB`(**v1.12**), `Tutorial`,
+  Valid values for this entry are `GameData`, `Ships`, `Ships/SPH`(**v1.12**), `Ships/VAB`(**v1.12**), `Tutorial`, `saves/scenarios` (**v1.14**),
```

Comment:
Oh dear, I've coded it as [just `Scenarios`](https://github.com/KSP-CKAN/CKAN/pull/1360/files#diff-742564cbb6ab84e72817e8b1bd7477a6R277).